tests/e2e/magic_functions/common.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 Memgraph Ltd.
```

Comment:
I don't think that is necessary for now, just as an idea for the future: as we are started to copy these functions, what about making a strictly internal library for testing utilities? That would be a nice way of unifying our e2e tests and the python code that they use.

@jbajic, as you are our relative python expert, how hard would this be?

Reply:
Actually, it would be fairly easy, since we can already use parts of gqlalchemy for our test, especially setup and teardown. I agree this part is not needed unless it would be used across all e2e tests (that are using python)